src/index.css

```diff
@@ -1,13 +1,31 @@
-body {
+* {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+  padding: 0;
+}
+
+body {
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
     sans-serif;
```

Comment:
- [ ] It would be nice if you could use Lato font as specified in the design guidelines for this project as required.

<img width="1390" alt="Screenshot 2023-02-09 at 1 32 41 AM" src="https://user-images.githubusercontent.com/50941074/217692392-9c177558-7cb5-4af9-bfd4-cfad28142f5b.png">